doc/bt/ZC_UPDATE_SHIELD.bt

```diff
@@ -0,0 +1,18 @@
+//------------------------------------------------
+//--- 010 Editor v8.0 Binary Template
+//
+//      File: ZC_UPDATE_SHIELD.bt
+//   Authors: celophi
+//   Version: i171140
+//   Purpose: Updates an actor's shield value.
```

Comment:
I should play some ToS some time, I still don't know what exactly this "shield" property is that I keep seeing in our code =P

Reply:
Me either! But, you should be able to find all of these in the lua files. That might give us a hint at what it does.